spec/yaml/swiftnav/sbp/ssr.yaml

```diff
@@ -435,7 +447,7 @@ definitions:
             desc: Array of STEC information for each space vehicle
 
  - MSG_SSR_GRIDDED_CORRECTION:
-    id: 0x05F0
+    id: 0x05FA
```

Comment:
@jbangelo @silverjam - should I re-order the messages so they are monotonically increasing in the yaml?  If there is another message with that ID, will the build step throw an error?

Reply:
if the answer is (no, yes), I am ready to push bindings

Reply:
pushed...

Reply:
I originally meant to deprecate the original message definition and add a new definition with the new fields. That would involve leaving the original definition in place, changing its name to be `MSG_SSR_GRIDDED_CORRECTION_DEP_A` and then adding the new message definition. That'll make sure that we don't end up reusing the same ID for something else down the line. It's a pretty remote chance of it happening but it could cause weird issues if it happens.

As for your question, I think one of the python tests will fail when it tries to store all the messages in a map by message ID and the map's length isn't what it expects.

Reply:
And I wouldn't worry too much about reordering the messages unless you've got free time on your hands 👍

Reply:
ah, yea - if I make a new message, certainly.  I remember there was some tool to print out all the IDs, no?
ok, I'll throw in the placeholder.